components/apimgt/org.wso2.carbon.apimgt.impl/src/test/java/org/wso2/carbon/apimgt/impl/utils/APIUtilTest.java

```diff
@@ -1946,7 +2038,429 @@ public void testIsRoleExistForUser() throws Exception {
     }
 
     @Test
-    public void testGetTokenEndpointsByType () throws Exception {
+    public void testHasPermissionWhenPermissionCheckDisabled() throws APIManagementException {
+
+        String userNameWithoutChange = "Drake";
+        String permission = APIConstants.Permissions.API_PUBLISH;
+
+        Log mockLog = Mockito.mock(Log.class);
+        PowerMockito.mockStatic(LogFactory.class);
+        Mockito.when(LogFactory.getLog(any(Class.class))).thenReturn(mockLog);
+
```

Comment:
Shall we remove this unnecessary empty lines?

Reply:
this is done